src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/BuildCardExtension/build-card-template-onclicks.js

```diff
@@ -0,0 +1,91 @@
+function parseDataAttributes(container) {
+    return {
+        id: parseInt(container.dataset.showSpinnerId, 10),
+        buildExtId: container.dataset.rerunBuildExtId,
+        dependencyIds: container.dataset.rerunBuildDependencyIds
+            .split(",")
+            .filter(Boolean)
+            .map(id => parseInt(id, 10))
+    };
+}
+
+function parseTriggerAttributes(container) {
+    return {
+        id: parseInt(container.dataset.showSpinnerId, 10),
+        upstreamProjectName: container.dataset.upstreamProjectName,
+        upstreamBuildNumber: parseInt(container.dataset.upstreamBuildNumber, 10),
+        projectName: container.dataset.projectName,
+        dependencyIds: container.dataset.dependencyIds
+            .split(",")
+            .filter(Boolean)
```

Comment:
What is this supposed to filter out? My guess would be spaces that may be after commas, but I'm not sure.
If that's the case though `Boolean(" ")` returns `true`.

Reply:
Just wanted to avoid any surprises here, but in further testing, ID's are being returned as a well formatted list.